.gitignore

```diff
@@ -16,7 +16,6 @@ build/
 
 # Gradle files
 .gradle/
-build/
```

Comment:
Some feedback about your commits: I'm trying to understand what kind of changes are needed with the current APIs to add support for a new markdown syntax but it's difficult to review them because your commits aren't atomic and include a lot of miscellaneous changes. Prefer splitting your commits into individual chunks so that they can be reviewed/skimmed faster.

Reply:
Oh ok, sorry for that. I have update some dependencies and used AndroidStudio formatting.

These files are the one I made main changes to add support for underline span
MarkdownHintStyles.kt, MarkdownSpanPool.kt, FlexmarkMarkdownParser.kt, FlexmarkNodeTreeVisitor.kt, HorizontalRuleSpan.kt.

Created new file UnderlineExtension.kt to create new FlexMarker node.

Reply:
Feature request, is it possible to have an option to toggle visibility of markdown syntax? 😂

Reply:
@msasikanth yea I guess including the code-style in the project would be useful.

> Feature request, is it possible to have an option to toggle visibility of markdown syntax? 😂. To have a clean edit text with formatted text

Interesting question. I assume others might also have this question. Could you please create a new issue for this for visibility?

Reply:
Absolutely 👍🏼